Exadel.Compreface.AcceptenceTests/Services/SubjectExampleServiceTest.cs

```diff
@@ -0,0 +1,254 @@
+﻿using Exadel.Compreface.Configuration;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.AddBase64ExampleSubject;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.AddExampleSubject;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.DeleteAllSubjectExamples;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.DeleteImageById;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.DeleteMultipleExamples;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.DownloadImageById;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.DownloadImageBySubjectId;
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.ListAllExampleSubject;
+using Exadel.Compreface.DTOs.HelperDTOs;
+using Exadel.Compreface.DTOs.SubjectDTOs.AddSubject;
+using Exadel.Compreface.DTOs.SubjectDTOs.DeleteSubject;
+using Exadel.Compreface.Exceptions;
+using static Exadel.Compreface.AcceptenceTests.UrlConstConfig;
+
+namespace Exadel.Compreface.AcceptenceTests.Services
+{
+    public class SubjectExampleServiceTest
+    {
+        private readonly ComprefaceClient comprefaceApiClient;
+        public SubjectExampleServiceTest()
+        {
+            comprefaceApiClient = new ComprefaceClient(new ComprefaceConfiguration(API_KEY_SUBJECT_SERVICE, BASE_URL));
+        }
+
+        [Fact]
+        public async Task AddSubjectExampleAsync_TakesRequestModel_ReturnsProperResponseModel()
+        {
+            //Arrange
+            var addNewSubjectResponse = await comprefaceApiClient.SubjectService.AddSubject(new AddSubjectRequest() { Subject = TEST_SUBJECT_NAME });
+            var subjectExample = new AddSubjectExampleRequest()
+            {
+                DetProbThreShold = 0.81m,
+                FilePath = FILE_PATH,
+                Subject = addNewSubjectResponse.Subject,
+                FileName = FILE_NAME
+            };
+
+            var expectedAddExampleSubjectResponse = await comprefaceApiClient.SubjectExampleService.AddSubjectExampleAsync(subjectExample);
```

Comment:
I think, If you're testing the AddSubjectExampleAsync method, what it returns is the actual value (not expected)... 
In this case, you've tested the GetAllSubjectExampleAsync method 😅
Here either the logic is wrong, or the name of the test method))))
But I don't know exactly... Maybe I'm just not good at testing 🤪